src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -2048,7 +2052,7 @@ public Template createTemplate(DataObject object)
         final VerificationLevel guildVerificationLevel = VerificationLevel.fromKey(guildObject.getInt("verification_level", -1));
         final NotificationLevel notificationLevel = NotificationLevel.fromKey(guildObject.getInt("default_message_notifications", 0));
         final ExplicitContentLevel explicitContentLevel = ExplicitContentLevel.fromKey(guildObject.getInt("explicit_content_filter", 0));
-        final Locale locale = Locale.forLanguageTag(guildObject.getString("preferred_locale", "en"));
+        final DiscordLocale locale = DiscordLocale.from(guildObject.getString("preferred_locale", "en-US"));
```

Comment:
This concerns me slightly that we had the default as `en` and now it is `en-US`. Has discord possibly been sending us `en-US` and did Locale#forLanguageTag know how to handle it? 🤔